challenges/classes.js

```diff
@@ -3,9 +3,47 @@
 // Test your volume and surfaceArea methods by uncommenting the logs below:
 // console.log(cuboid.volume()); // 100
 // console.log(cuboid.surfaceArea()); // 130
-
+class cuboidMaker{
+  constructor(dimensions){
+    this.length = dimensions.length;
+    this.width = dimensions.width;
+    this.height = dimensions.height;
+  }
+  volume(length,width,height){
+    return(this.length * this.width * this.height);
+  }
+  surfaceArea(length,width,height){
+    return(this.length * this.width + this.length* this.height + this.width * this.height);
```

Comment:
need to multiply this by 2..., there are 6 sides on a cuboid, and you added together the surface area from 3 squares